changelogs/fragments/firewalld_zone_target.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - firewalld - Bring the ``target`` feature back (Was deleted during the migration from community.general).
```

https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment
> Each changelog entry must contain a link to its issue between parentheses at the end. If there is no corresponding issue, the entry must contain a link to the PR itself.
```suggestion
  - firewalld - Bring the ``target`` feature back (https://github.com/ansible-collections/ansible.posix/issues/112).
```

---

Done